### PR TITLE
Update setupchecks.js - Changed wording of message

### PR DESCRIPTION
**Pull request by rakekniven**
Changed wording of setup check.
Reported at Transifex.

Signed-off-by: rakekniven <mark.ziegler@rakekniven.de>

### CODE REVIEW

File: core/js/setupchecks.js [hunk 1]
```diff
@@ -389,7 +389,7 @@
 						messages.push({
 							msg: t(
 								'core',
-								'Module php-imagick in instance is missing SVG support. For better compatibility it is recommended to install it.'
+								'Module php-imagick in this instance has not SVG support. For better compatibility it is recommended to install it.'
```
Comment (skjnldsv) [inline core/js/setupchecks.js:392]:
```suggestion
								'Module php-imagick in this instance has no SVG support. For better compatibility it is recommended to install it.'
```

A bit better?

**Comment by skjnldsv** [comment]
@rakekniven please add labels to your tickets.

**Review by skjnldsv — APPROVED** [review]

**Review by GretaD — APPROVED** [review]

**Comment by MorrisJobke** [comment]
no backport needed - was added to 21: #23677

**Pull request merged**